### ISSUE TITLE
tailcall_v4\hijacking\hijacking.cmd fails in arm64

### ISSUE THREAD

**Issue by ramarag** [dotnet/coreclr#4122]
BEGIN EXECUTION
 "Z:\arm64\new\corerun.exe" hijacking.exe
Executed 34 GCs
Executed 6 tail calls before failing
Expected: 100
Actual: 101
END EXECUTION - FAILED
FAILED

@briansull  please investigate\assign this to the proper dev

**Comment by rartemev** [comment]
How much time does it usually take to be failed? I cannot reproduce the bug because execution is hanging on right after invoking "Thread.Sleep()" from Main thread.

**Comment by jashook** [comment]
Currently this test will not fail. It will run indefinitely.

**Comment by RussKeldorph** [comment]
Right "failure" means "test runner times out" in this case.  Sorry if that wasn't clear.

**Comment by sdmaclea** [comment]
This appears fixed by #12047

**Comment by rartemev** [comment]
The original problem with fully interruptible code is fixed by #12047 so there is no reason left to hang on.

```
C:\Users\rartemev>timecmd.bat C:\Users\rartemev\Desktop\Tests\Core_Root\Corerun.exe C:\Users\rartemev\Desktop\Tests\tailcall_v4\hijacking\hijacking.exe
Executed 31269 GCs
Executed 2105195275 tail calls
PASS
command took 0:1:11.29 (71.29s total)
```

The average execution time is about 70 seconds.
